bin/build/build.sh

```diff
@@ -1,19 +1,51 @@
 #!/bin/bash
 
+declare htaccess_config_default="htaccess.conf";
+declare htaccess_output_default="./.htaccess"
+declare repo_root="$(cd "$(dirname "$0")" && cd ../../ && pwd)"
+declare temp_directory
+declare htaccess_output_tmp
+
+# ----------------------------------------------------------------------
+# | File system                                                        |
+# ----------------------------------------------------------------------
+
+prepareTempDirectory() {
+    temp_directory="$(mktemp -d ${TMPDIR:-/tmp/}server-configs-apache.XXXXXXXXXXXX)"
+    htaccess_output_tmp="${temp_directory}/htaccess.tmp"
+
+    if [ ! -d "${temp_directory}" ]; then
+        print_error "Creating temp directory failed."
+        exit 1
+    fi
+}
+
+# Default Exit or SIGINT(2) handler
+trapCleanupTempDir() {
+    [ -d "${temp_directory}" ] \
+    && rm -Rf "${temp_directory}" \
+    && print_success "Tidy up filesystem"
+}
+
+trap trapCleanupTempDir EXIT SIGINT
+
 # ----------------------------------------------------------------------
 # | Helper functions                                                   |
 # ----------------------------------------------------------------------
 
 clean() {
-    rm -rf dist/
-    rm -rf test/fixtures/.htaccess
+    for removee in "$@"
+    do
+        rm -rf "${removee}"
```

Comment:
Actually recursive flag can be removed here, I think. It would avoid dumb user to delete the whole system! 🤣

Reply:
Good one. It's not used anyway. To be removed.